packages/admin-ui/src/button/button.style.ts

```diff
@@ -116,6 +116,7 @@ export const variants = styleVariants({
 
 export const innerContainerStyle = style({
   text: '$action1',
+  whiteSpace: 'nowrap',
```

Comment:
This fixes the problem of text wrapping when the button label has a white space.

**Before**
<img width="365" alt="Captura de Tela 2023-03-17 às 10 49 59" src="https://user-images.githubusercontent.com/20579226/225923741-4d12e303-918e-4c57-88f9-be7a48a66811.png">

**After**
<img width="379" alt="Captura de Tela 2023-03-17 às 10 47 59" src="https://user-images.githubusercontent.com/20579226/225923254-2a5e3bce-1003-4056-83cb-39f65f35601b.png">